src/components/NumberButton.js

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+import {NumberColor} from "../constants/GameConstants";
+
+const NumberButton = props => (
+    <button
+        className="number"
+        style={{backgroundColor: NumberColor[props.status]}}
+        onClick={() => props.onClick(props.number)}>
```

Comment:
small performance issue. Should avoid arrow functions in render, if possible
https://reactjs.org/docs/faq-functions.html#arrow-function-in-render